File: Programmers/정렬_K번째수.java
import java.util.*;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        
        for(int p=0; p<commands.length; p++){
            int i = commands[p][0];
            int j = commands[p][1];
            int k = commands[p][2];
            int[] cut = new int[j-i+1]; //자른 배열

            for(int q=i-1;q<j;q++){
                cut[q-i+1] = array[q];   
            }

            Arrays.sort(cut);
            answer[p] = cut[k-1];
        }
        
        
        return answer;
    }
}

File: Programmers/정렬_K번째수.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    vector<int> answer;
    
    for(auto c: commands){
        vector<int> v;
        for(int i=c[0]-1; i<c[1]; i++){
            v.push_back(array[i]);
        }
        sort(v.begin(), v.end());
        answer.push_back(v[c[2]-1]);
    }
    
    return answer;
}